no-publish/src/main/scala/org/typelevel/sbt/NoPublishPlugin.scala

```diff
@@ -40,17 +46,14 @@ object NoPublishGlobalPlugin extends AutoPlugin {
   override def trigger = allRequirements
 
   object autoImport {
-    lazy val noPublishProjectRefs = settingKey[Seq[ProjectRef]]("List of no-publish projects")
+    private[sbt] lazy val noPublishModulesIgnore =
```

Comment:
Oh, sorry one last thing: since this is `private` there's no point to put it in the `autoImport`, since nobody can use it from `build.sbt` anyway. We can just keep it in the object.

Reply:
fair, I wasn't sure if that was to just keep the imports cleaner in the other parts, I'll change that right away :)